Library/Source/XMLHttpRequest.cpp

```diff
@@ -130,8 +130,19 @@ namespace Babylon
     {
         if (m_responseType.empty() || m_responseType == XMLHttpRequestTypes::ResponseType::Text)
         {
-            return m_runtimeImpl.LoadUrlAsync<std::string>(m_url).then(arcana::inline_scheduler, m_runtimeImpl.Cancellation(), [this](const std::string& data) {
-                m_responseText = std::move(data);
+            return m_runtimeImpl.LoadUrlAsync<std::string>(m_url).then(arcana::inline_scheduler, m_runtimeImpl.Cancellation(), [this](const std::string& data)
+            {
+                // check UTF-8 BOM encoding
```

Comment:
Do we only need to remove UTF-8 BOM? Are there other BOMs we need to consider?

Reply:
I don't know! It's the first case I've found and tested. I don't know if there are other BOM, nor how I can test it. If you have some doc on that, I'd be happy to learn more.

Reply:
I think LoadUrlAsync should be doing binary to text conversion. Looking at the implementation, my guess is that cURL doesn't do binary to string conversion after getting the data from the network. What if the data is Unicode or some other encoding?

Reply:
I added an item to issue #96 for this.